egs/fisher_swbd/s5/local/nnet3/run_tdnn_discriminative.sh

```diff
@@ -0,0 +1,188 @@
+#!/bin/bash
```

Comment:
Sure, I will review this tonight.

> --
Vimal Manohar
PhD Student
Electrical & Computer Engineering
Johns Hopkins University